help/destinations/catalog/email-marketing/mailchimp-interest-categories.md

```diff
@@ -0,0 +1,239 @@
+    
+    The mappings available for the `ADDRESS` field are as below:
+
+    | Source Field | Target Field |
+    | --- | --- |
+    | `xdm: workAddress.street1` | `Attribute: ADDRESS.addr1` |
+    | `xdm: workAddress.street2` | `Attribute: ADDRESS.addr2` |
+    | `xdm: workAddress.city` | `Attribute: ADDRESS.city` |
+    | `xdm: workAddress.state` | `Attribute: ADDRESS.state` |
+    | `xdm: workAddress.postalCode` | `Attribute: ADDRESS.zip` |
+    | `xdm: workAddress.country` | `Attribute: ADDRESS.country` |
+
+    For example, you want to update the value for `country` with the contact's existing address field `addr1`, `city`, `state`, and `zip` values as `132, My Street, Kingston`, `New York`, `New York` and `12401`. To update the `country` you need to pass the existing values with changes *(if any)* and the new value for country. So the values in your dataset should be `132, My Street, Kingston`, `New York`, `New York`, `12401` and `US`. To reiterate, if you only pass `country` and do not provide values for `addr1`, `city`, `state`, and `zip` they will be overwritten by `NULL`.
+
+    An example with the completed mappings is shown below:
+    ![Platform UI screenshot example showing field mappings.](../../assets/catalog/email-marketing/mailchimp-interest-categories/mappings.png)
+
+When you have finished providing the mappings for your destination connection, select **[!UICONTROL Next]**.
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+* Log in to your [[!DNL Mailchimp]](https://login.mailchimp.com/) account. Then, navigate to the **[!DNL Audience]** page. Next, expand the **[!DNL Manage Contacts]** menu and select **[!DNL Groups]**.
+
+![Mailchimp UI screenshot showing the Audience group page.](../../assets/catalog/email-marketing/mailchimp-interest-categories/audience-groups.png)
+
+* Select the Group and check if the selected segments are created as categories with the segment name from Platform, which may be followed by an automatically generated suffix.
+  * This destination uses the selected segments' names to create the interest category by leveraging the [[!DNL Mailchimp] Add interest category API](https://mailchimp.com/developer/marketing/api/interest-categories/add-interest-category/). If you create a new destination and activate the same segments, because the selected segments' names already exist in [!DNL Mailchimp] the destination adds a suffix to distinguish between the segments.
```

Comment:
comma missing:

If you create a new destination and activate the same segments, because the selected segments' names already exist in [!DNL Mailchimp], the destination adds a suffix to distinguish between the segments.